plugins/include/sqlx.inc

```diff
@@ -30,33 +30,48 @@ enum Handle
 };
 
 /**
- * Creates a connection information tuple.
- * This tuple must be passed into connection routines.
- * Freeing the tuple is not necessary, but is a good idea if you 
- *  create many of them.  You can cache these handles globally.
- * !!NOTE!! I have seen most people think that this connects to the DB.
- *   Nowhere does it say this, and in fact it does not.  It only caches
- *   the connection information, the host/user/pass/etc.
- *
- * The optional timeout parameter specifies how long connections should wait before
- * giving up.  If 0, the default (which is undefined) is used.
- *
+ * Creates a connection information tuple. This tuple must be passed
+ * into connection routines.
+ * 
+ * @note Freeing the tuple is not necessary, but is a good idea if you create
+ *       many of them. You can cache these handles globally.
+ * @note This does not connect to the DB; it only caches the connection information.
+ * 
+ * @param host          Database host
+ * @param user          Database user
+ * @param pass          Database password
+ * @param db            Database name to use
+ * @param timeout       Specifies how long connections should wait before giving up.
+ *                      If 0, the default (which is undefined) is used.
```

Comment:
If provided timeout is <= 0 (whatever from native or core.ini), mysql will set the default timeout to 60s.